pkg/mic/vmss.go

```diff
@@ -96,6 +96,11 @@ func makeVMSSID(r azure.Resource) string {
 	return path.Join(r.SubscriptionID, r.ResourceGroup, r.ResourceName)
 }
 
+func getVMSSName(vmssID string) string {
+	_, resourceName := path.Split(vmssID)
```

Comment:
Is it possible to have an erroneous vmssID, if so how do we handle this ?

Reply:
If the vmssID is erroneous, then we fail while trying to create the client and not perform any operations.

Reply:
Can we please handle the error for the path.Split returning an empty string ?